swap_meet/clothing.py

```diff
@@ -1,2 +1,7 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
+    def __init__(self, condition = 0, age = 0):
```

Comment:
Nice work removing `category` from the available initializer parameters. Doing so indicates that we don't want the caller to be able to set a different category.

The caller _could_ always change the category by assigning directly to it after:
```python
clothing = Clothing(condition=3)
clothing.category = "Electronics"
```

But at least we're indicating that they shouldn't do that when we organize the code this way. 

Also, with default params we tend to leave spaces off from each side of the equal sign:
```python
def __init__(self, condition=0, age=0):